extensions/mssql/src/credentialstore/credentialstore.ts

```diff
@@ -70,7 +70,7 @@ export class CredentialStore {
 	private _client: SqlOpsDataClient;
 	private _config: IConfig;
 
-	constructor(baseConfig: IConfig) {
+	constructor(private logPath: string, baseConfig: IConfig) {
```

Comment:
nit: should this be called _logPath?

Reply:
if you do not waht to change to _logPath then consider renaming existing private fields _client and _config to not start with _ for consistency.